react/README.md

```diff
@@ -202,8 +290,8 @@
   // good
   <Foo className="stuff"/>
```

Comment:
update this to have a space at the end `<Foo className="stuff" />`

Reply:
yup, good catch